docs/logs/configuring.asciidoc

```diff
@@ -0,0 +1,105 @@
+[role="xpack"]
+[[xpack-logs-configuring]]
+
+:ecs-link: {ecs-ref}[Elastic Common Schema (ECS)]
+
+== Configuring the Logs UI
+
+The `filebeat-*` index pattern is used to query data by default. If your logs
+are located in a different set of indices, use a different timestamp field, or
+contain parsed fields which you want to expose as individual columns, you can
+adjust the source configuration via the user interface or the {kib}
+configuration file.
+
+NOTE: Logs and Infrastructure share a common data source definition in
+each space. Changes in one of them can influence the data displayed in the
+other.
+
+[float]
+=== Configure source
+
+Configure source can be accessed via the corresponding
+image:logs/images/logs-configure-source-gear-icon.png[Configure source icon]
+button in the toolbar.
+
+[role="screenshot"]
+image::logs/images/logs-configure-source.png[Configure Logs UI source button in Kibana]
+
+This opens the source configuration fly-out dialog with multiple tabs, on which
+various index settings and the log column configuration can be inspected and
+adjusted.
+
+TIP: If <<xpack-spaces>> are enabled in your Kibana instance, any configuration
+changes performed via Configure source are specific to that space. You can
+therefore easily make different subsets of the data available by creating
+multiple spaces with different data source configurations.
+
+[float]
+[[logs-read-only-access]]
+==== Read only access
+When you have insufficient privileges to change the source configuration, the following
+indicator in Kibana will be displayed. The buttons to change the source configuration
+won't be visible. For more information on granting access to
+Kibana see <<xpack-security-authorization>>.
```

Comment:
(missing comma)
>on granting access to Kibana, see

Reply:
I'm curious to learn what the grammer rule for that would be. As a non-native speaker a comma would feel wrong here.

Reply:
I would change this paragraph to something like:
```asciidoctor
When you have insufficient privileges to change the source configuration, the following
indicator in Kibana will be displayed, and the buttons to change the source configuration
won't be visible. For more information, see <<xpack-security-authorization>>.
```

Reply:
Thanks for the suggestion, will do! Just as a note: this snippet is repeated all across the Kibana docs wherever there is a special UI behavior with read-only privileges.